scenario/context/CometContext.ts

```diff
@@ -271,7 +272,7 @@ async function buildActor(name: string, signer: SignerWithAddress, context: Come
 const getInitialContext = async (world: World): Promise<CometContext> => {
   let deploymentManager = new DeploymentManager(world.base.name, world.hre, { debug: true });
 
-  if (!world.isRemoteFork()) {
+  if (!world.isRemoteFork() || world.base.deployNewComet) {
```

Comment:
Do we need this option? Jared's changes will automatically deploy a new Comet for instances where a `roots.json` is either empty or incomplete.